src/globals.c

```diff
@@ -212,13 +213,37 @@ bool set_format_string(char *userinput)
 	return true;
 }
 
+bool set_state_dir(char *dir)
```

Comment:
done.

On Mon, Apr 11, 2016 at 4:18 PM, Jaime Garcia <fandeli@gmail.com> wrote:

> all right
>
> On Mon, Apr 11, 2016 at 3:57 PM, Patrick McCarty <notifications@github.com
> > wrote:
>
>> In src/globals.c
>> <https://github.com/clearlinux/swupd-client/pull/35#discussion_r59280093>
>> :
>>
>> > @@ -212,13 +213,37 @@ bool set_format_string(char *userinput)
>> >  	return true;
>> >  }
>> >
>> > +bool set_state_dir(char *dir)
>>
>> In init_globals(), I would like to call set_state_dir(NULL), but that's
>> not possible in the current state, because dir==NULL will return false. I'm
>> suggesting to make NULL a special value, just like it is for the other
>> set_* functions.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/clearlinux/swupd-client/pull/35/files/4bbe7cda9873dd196d837219bef94f800ec1a1e5#r59280093>
>>
>
>